### ISSUE TITLE
Can we send the token from Angular UI application to ssrs 

### ISSUE THREAD

**Issue by karthiktanj** [microsoft/Reporting-Services#179]
I need to authenticate the token from Angular application to SSRS . Could you please anyone suggest is it possible in SSRS 2014 and SSRS 2016
Scenario 
UI Application is Angular 8. User is login into the Enterprise Portal, once login the usename is validated and token will be generated . There is multiples plugins in the Enterprise Portal, once the user click the Report Plugin  already generated token will be in the angular session and it will send to SSRS Portal along with URL 
Syntax: http://hostname or IP Address/Reports/token
Example : http://localhost/Reports/asdfghjkiouytrer

Could you please anyone help me the EndUser  token can be access in the Report Server with custom authentication without using any use accounts database. The token send to api server to authenticate . If the api server is return message with user it will be successfully authenticated or it returns null value the user is a invalid user. This is for security purpose anyone try to access the report server url directly the report server page has to be return login page.

Any one suggest can we do  the above functionality

**Comment by Amethi** [comment]
Yes, you could use this sample to make SSRS token-enabled. Strip out the database functionality and implement a token-based approach in the Logon.aspx page. You might not be able to use the same token from the Angular app but you can have it request a token from your IDP so the user is logged in to SSRS using tokens.

**Comment by pveldi** [comment]
> Yes, you could use this sample to make SSRS token-enabled. Strip out the database functionality in the custom security example and implement a token-based approach in the Logon.aspx page. You might not be able to use the same token from the Angular app but you can have it request a token from your IDP so the user is logged in to SSRS using tokens (SAML/OIDC, etc.).

Hi Amethi,
Do you have any working example of this?

Thanks
